eng/native/functions.cmake

```diff
@@ -331,7 +285,16 @@ function(strip_symbols targetName outputFilename)
 
     set(${outputFilename} ${strip_destination_file} PARENT_SCOPE)
   else(CLR_CMAKE_HOST_UNIX)
-    set(${outputFilename} ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/${targetName}.pdb PARENT_SCOPE)
+    get_property(_isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+    if(_isMultiConfig)
+      # We can't use the $<TARGET_PDB_FILE> generator expression here since
```

Comment:
Yes, when I first wrote this, we end up coming down this path for the resource dlls. It's possible with the numerous refactorings since then that this has changed.

Reply:
Yep we still call `strip_symbols` for resource dlls.